tags_test.go

```diff
@@ -152,8 +152,8 @@ func TestFindTags(t *testing.T) {
 	}
 
 	res, err = sc.FindTags(1, "test", &FindTagsOptions{
-		Start:     time.Unix(1, 0),
-		End:       time.Unix(2, 0),
+		StartStr:  "now-1d",
+		EndStr:    "now",
```

Comment:
Spaces are also legal, so you should add "now - 1d" as well

Reply:
Good catch. This would have resulted in URL errors.